impl/client/src/doc/javascript/template/vizApi/tmpl/details.tmpl

```diff
@@ -54,7 +54,7 @@ if (data.defaultvalue && (data.defaultvaluetype === 'object' || data.defaultvalu
 </ul></dd>
 <?js } ?>
 
-<?js if (data.implements && data.implements.length) { ?>
+<?js if (data.kind !== "class" && data.implements && data.implements.length) { ?>
```

Comment:
Don't see what other kind would want this here and not at the new place. Interfaces want it at the top also. What's left?